src/test/java/com/google/api/codegen/testdata/go_library.baseline

```diff
@@ -635,54 +635,54 @@ func (it *StringIterator) NextPageToken() string {
     return it.nextPageToken
 }
 ============== file: google.golang.org/cloud/library/generated/library/client_test.go ==============
-package library
+package library_test
 
 import (
     gax "github.com/googleapis/gax-golang"
     "golang.org/x/net/context"
     google_example_library_v1 "google.golang.org/cloud/library/generated/google/example/library/v1"
```

Comment:
I'm not sure how protobuf compiler chooses the name. However, if the proto package is called "library", we probably shouldn't also call the client library "library" unless we want to put the two in the same package.